src/test/java/org/eolang/speco/SpecoTest.java

```diff
@@ -58,6 +58,7 @@ class SpecoTest {
     private static final int INTENT = 11;
 
     @Tag("fast")
+    @Disabled
```

Comment:
Why not `@DisabledOnOs(OS.WINDOWS)`?

Reply:
@volodya-lombrozo Because rule 3 requires major changes in both `xmir` and `eo` output formats and the transformation itself will be added in the next PR, so all tests expected fall.